web/crux/.env.example

```diff
@@ -60,8 +60,8 @@ DISABLE_RECAPTCHA=true
 # Required only when ReCAPTCHA is enabled
 RECAPTCHA_SECRET_KEY=<recaptcha_secret_key>
 
-# Determines the quantity of lines being returned by a Docker container, while streaming its logs
-DEFAULT_CONTAINER_LOG_TAIL=100
+# Determines the maximum quantity of lines returned by a container
```

Comment:
The phrasing is incorrect, the logs are not returned "by" the container. It could either be "from" or "of".